libraries/lib-time-and-pitch/StaffPadTimeAndPitch.cpp

```diff
@@ -103,7 +104,22 @@ void StaffPadTimeAndPitch::GetSamples(float* const* output, size_t outputLen)
    }
 }
 
-bool StaffPadTimeAndPitch::_SetRatio(
+bool StaffPadTimeAndPitch::CanReturnMoreSamples() const
+{
+   return !mAudioSource.Empty() ||
+          (mTimeAndPitch &&
+           mTimeAndPitch->getNumAvailableOutputSamples() > mNumTrailingZeros);
```

Comment:
Just added the following comment (not pushed yet)
```cpp
   // If our audio source isn't empty, we'll have samples to feed
   // `mTimeAndPitch` with, who in turn will yield some audio.- But it can also
   // be that our source is empty, but `mTimeAndPitch` still has samples in its
   // ISTFT buffer. We don't want to miss those ...
```